src/api_server.cc

```diff
@@ -155,7 +155,7 @@ MetadataReporter::MetadataReporter(MetadataAgent* agent, double period_s)
       environment_(agent->config_),
       auth_(environment_),
       period_(period_s),
-      reporter_thread_(&MetadataReporter::ReportMetadata, this) {}
+      reporter_thread_([=]() { ReportMetadata(); }) {}
```

Comment:
It can, and probably should in the future. However, #74 used `[=]` consistently throughout, so I'm just following that style. If we decide to tighten these captures, we could do it in a separate cleanup (and at this point I'm not sure it's worth it).